src/components/Common/Header/Header.jsx

```diff
@@ -0,0 +1,49 @@
+import propTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import style from './Header.module.css';
+import logo from '../../../assets/img/logo.png';
+import { Logout } from '../../Pages/Logout/Logout';
+
+export function Header({ userEmail, handleLogout }) {
+  return (
+    <header className={style.header}>
+      <div className={style.contentWrapper}>
+        <NavLink to='/' activeClassName='selected'>
+          <img className={style.logo} src={logo} alt='logo' />
+        </NavLink>
+        <nav>
+          <ul className={style.headerNav}>
+            <li className={style.navItem}>
+              <NavLink to='/members' activeClassName={style.selected}>
+                Members
+              </NavLink>
+            </li>
+            <li className={style.navItem}>
+              <NavLink to='/tasks' activeClassName={style.selected}>
+                Tasks
+              </NavLink>
+            </li>
+          </ul>
+        </nav>
+        <nav className={style.login}>
+          <ul>
+            <li>
+              <NavLink to='/login' activeClassName='selected'>
+                {userEmail ? <Logout handleLogout={handleLogout} /> : `Login`}
+              </NavLink>
```

Comment:
it can be rewritten by using `map` if you use nav links as items. please fix it )

Reply:
```suggestion
            {LINKS_HEADER_MENU.map((item) => (
              <NavLink
                key={item.name}
                to={item.to}
                className={style[item.className]}
                activeClassName={style[item.activeClassName]}
              >
                {item.name}
              </NavLink>
            ))}
```